public/wp-content/themes/glific/page-contact.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Page contact for theme glific
+ *
+ * @package glific
+ */
+
+	get_header();
+?>
+<div class="page-contact bg-theme-white-smoke pb-26">
+	<div class="pt-10 pt-md-16 pt-xl-18 pb-xl-4 pb-md-14 pb-10">
+		<h3 class="text-theme-primary font-heebo-bold fz-28 fz-md-36 leading-40 mb-0 text-center mb-6 mb-md-14 mb-xl-18"><?php echo get_field('contact_page_title'); ?></h3>
+	</div>
+	<div class="contact-form bg-white w-90p w-md-717 w-xl-805 mx-auto rounded-30 p-8">
+		<p class="mb-6 fz-18 leading-27 font-heebo-regular text-theme-mine-shaft text-theme-mine-shaft">
+			<?php echo get_field('contact_page_tagline'); ?>
+		</p>
+		<?php if (is_active_sidebar('glific_form_contact')) : ?>
+			<div class="contact-form">
+				<div class="form-group mb-md-0 list-unstyled">
+					<div id="glific_form_contact" role="complementary">
+						<?php dynamic_sidebar('glific_form_contact'); ?>
+					</div>
+				</div>
+			</div>
+		<?php endif; ?>
+	</div>
+	<?php 
+		$get_help_section = get_field('contact_get_help_section'); 
+	?>
+	<div class="get-help-quickly py-26">
+		<h3 class="text-theme-primary font-heebo-bold fz-28 fz-md-36 leading-40 mb-0 text-center mb-6 mb-md-14 mb-xl-10"><?php echo $get_help_section['contact_get_help_title']; ?></h3>
```

Comment:
I see you are fetching the `$get_help_section` value in different places. This will break in case the array is empty. Can we put a check at line 31 something like to ensure it will not break.  
`if(!empty($get_help_section)) { ?>`.

Reply:
👍